### PR TITLE
James 2292 webadmin mailqueues

### PR DESCRIPTION
**Pull request by mbaechler**
`GET /mailqueues` part

### CODE REVIEW

File: server/protocols/webadmin/webadmin-mailqueue/pom.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <groupId>org.apache.james</groupId>
+        <artifactId>james-server</artifactId>
+        <version>3.1.0-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>james-server-webadmin-mailqueue</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Apache James :: Server :: Web Admin :: mailbox</name>
```
Comment (rouazana) [inline server/protocols/webadmin/webadmin-mailqueue/pom.xml]: mailqueue (copy/paste detected)

File: server/protocols/webadmin/webadmin-mailqueue/pom.xml [hunk 2]
```diff
@@ -30,9 +30,42 @@
     <artifactId>james-server-webadmin-mailqueue</artifactId>
     <packaging>jar</packaging>
 
-    <name>Apache James :: Server :: Web Admin :: mailbox</name>
+    <name>Apache James :: Server :: Web Admin :: MailQueue</name>
```
Comment (rouazana) [inline server/protocols/webadmin/webadmin-mailqueue/pom.xml]: oh fixed

**Comment by mbaechler** [comment]
I reworked the history to fix raphael's comment

File: server/protocols/webadmin/webadmin-mailqueue/src/test/java/org/apache/james/webadmin/routes/MailQueueRoutesTest.java [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    }
+
+    @After
+    public void tearDown() {
+        webAdminServer.destroy();
+    }
+
+    @Test
+    public void listAllMailQueuesShouldReturnEmptyWhenNone() {
+        List<String> actual = RestAssured.when()
```
Comment (chibenwa) [inline server/protocols/webadmin/webadmin-mailqueue/src/test/java/org/apache/james/webadmin/routes/MailQueueRoutesTest.java]: Static imports?

Comment (chibenwa) [reply server/protocols/webadmin/webadmin-mailqueue/src/test/java/org/apache/james/webadmin/routes/MailQueueRoutesTest.java]: Fixed in https://github.com/linagora/james-project/pull/1257

**Comment by aduprat** [comment]
test this please

**Comment by chibenwa** [comment]
Merged. See https://github.com/linagora/james-project/pull/1258

**Pull request closed without merging**